pkg/util/util_test.go

```diff
@@ -332,3 +334,383 @@ func TestJoinQualifiedName(t *testing.T) {
 		}
 	}
 }
+
+const gatewayfirst = `Iface	Destination	Gateway 	Flags	RefCnt	Use	Metric	Mask		MTU	Window	IRTT                                                       
```

Comment:
General note about table-driven tests below.  Ideally the details of each test case are lumped together, so the []net.Addr that goes with each of these is defined right next to this table, and that is defined right next to the code.